src/scss/linkList.scss

```diff
@@ -174,5 +176,23 @@
       height: 1em;
     }
 
+    &-disabled {
+      right: 0;
+      top: 0;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 50px;
+      height: 100%;
+      background-color: $color-light-blue;
+      color: $color-primary;
+      margin-right: 5px;
```

Comment:
This should not be needed as it is a copy of the parent `.linkButton`.
Only the opacity and cursor changes needed.
Perhaps we can change to cursor to `cursor: not-allowed` to give another hint to the user, that he can't click here instead of `default`.